src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#laptops&computer"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
```

Comment:
add an empty line between multiline siblings
```suggestion
          <li class="nav__item">
            <a href="#photo" class="nav__link">Photo</a>
          </li>
          
          <li class="nav__item">
            <a href="#video" class="nav__link">Video</a>
          </li>
```